### PR TITLE
Save workflow

### LINKED ISSUE #81: Additional instruction text / tutorial regarding how to save data
### Request:
I think we probably need clearer instructions for users about how to save their data. If you both agree, where should we put this? Maybe the last page, where results can be downloaded? Before?

### Feature Description:
We will add a prompt to indicate to users that they may want to save their data, the prompt will include an option to save the data which will export the data as a .JSON file. The prompt will disappear a_er 60 seconds. We will also adjust “export” to be “save” data until the final results and advocate page which will remain “export”.

### PR DESCRIPTION
**Pull request by sschafft**
PR to support save data reminders.

Resolves #81 

### Save Reminder Text
image

### Automated save reminder announcement (with link to export)
image

### Text changes from export to save
Save on all pages except Results & Advocate
image

Export on Results & Advocate
image

### CODE REVIEW

**Comment by sschafft** [comment]
@ewilliams3794 - the save workflow is finished. Feel free to review any of the text changes / let me know if you want to tweak any of that.

File: src/locales/en.yaml [hunk 1]
```diff
@@ -60,6 +61,7 @@ en:
     restoreActivities: Restore your activities
     addActivities: Add your Activites
     editActivities: Edit your Activites
+    saveDataReminder: 'Your web browser will automatically keep your data present. However, **this is not a gurantee that your data will always be saved.** We recommend you save your data manually on a regular basis to make sure you dont lose any data. There are many prompts to save your data and you can click [here](%{exportLink}) to save your data.'
```
Comment (hcline) [inline src/locales/en.yaml]: dont --> don't

Comment (hcline) [inline src/locales/en.yaml]: dont --> don't

**Comment by sschafft** [comment]
@hcline - made the changes requested. can you approve?

**Pull request merged**